cmd/erda-server/bootstrap.yaml

```diff
@@ -81,9 +81,11 @@ erda.core.dicehub.image: { }
 erda.core.dicehub.release:
   max_time_reserved: "${RELEASE_MAX_TIME_RESERVED:72}"
   gc_switch: "${RELEASE_GC_SWITCH:true}"
-erda.core.dicehub.extension:
+extension:
```

Comment:
why there are two extension providers?

Reply:
`extension` is new public provider

Reply:
Unclear. You should declare the difference between `extension` and `erda.core.dicehub.extension`.